lib/Queue.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Queue => Stack
+ * -----------------------------
+ * Queue a stack data structure.
+ *
```

Comment:
describe that there should be a constructor method, and the constructor arguments (_none_)